packages/components/button/examples/ButtonGroupCollapsedExample.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Stack,
+  ButtonGroup,
+  Button,
+  IconButton,
+} from '@contentful/f36-components';
+import { ChevronDownIcon, PlusIcon } from '@contentful/f36-icons';
+
+export default function ButtonGroupCollapsedExample() {
+  return (
+    <Stack>
```

Comment:
```suggestion
    <Stack flexDirection="column">
```

Since this two groups are different examples (I'm assuming this because of the text before them)
I would split them or show them in a column otherwise it feels to me that they are part of the same "thing"

Reply:
👍🏼